include/__threading_support

```diff
@@ -93,8 +95,13 @@ typedef void* __libcpp_recursive_mutex_t[5];
 #endif
 
 // Condition Variable
+#if defined(NXDK)
+typedef void* __libcpp_condvar_t[4];
```

Comment:
Where does the 4 come from?

Reply:
Added a comment here, too.

Reply:
nit: I'd prefer "nxdk" in comment over NXDK.
(other instance, too)

Reply:
Changed.